android/src/main/java/io/radar/react/RNRadarModule.java

```diff
@@ -215,15 +252,18 @@ public void trackOnce(ReadableMap locationMap, final Promise promise) {
 
         double latitude = locationMap.getDouble("latitude");
         double longitude = locationMap.getDouble("longitude");
-        float accuracy = (float)locationMap.getDouble("accuracy");
+        float accuracy = (float) locationMap.getDouble("accuracy");
         Location location = new Location("RNRadarModule");
         location.setLatitude(latitude);
         location.setLongitude(longitude);
         location.setAccuracy(accuracy);
 
         Radar.trackOnce(location, new Radar.RadarTrackCallback() {
             @Override
-            public void onComplete(@NonNull Radar.RadarStatus status, @Nullable Location location, @Nullable RadarEvent[] events, @Nullable RadarUser user) {
+            public void onComplete(@NonNull Radar.RadarStatus status,
+                                   @Nullable Location location1,
```

Comment:
Why the change to `location1`?

Reply:
This occurred during some auto-refactors. I had tried using lambdas instead of callbacks in this module, however because the example app targets a lower android build tools version/AGP, lambdas aren't supported when consumed, so I changed back. As a result, some of these names changed. I just changed this one back - please lmk if you find others.